### PR TITLE
Add 'keybase wallet init'

### PR DESCRIPTION
**Pull request by mlsteele**
Add `keybase wallet init` which creates a users initial wallet bundle and posts with a sigchain link. It's a dev only command. In the future this will run in the background.

### CODE REVIEW

File: go/client/cmd_wallet.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ func newCmdWallet(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Comman
 	subcommands := []cli.Command{
 		newCmdWalletBalances(cl, g),
 	}
-
+	subcommands = append(subcommands, getBuildSpecificWalletCommands(cl, g)...)
```
Comment (patrickxb) [inline go/client/cmd_wallet.go]: should we put balances in here too?

**Comment by patrickxb** [comment]
What happens if you run it twice?

File: go/client/cmd_wallet.go (hunk 1 above)
Comment (patrickxb) [reply go/client/cmd_wallet.go]: nvm...you're saying `init` will always be dev-only.  got it.

**Comment by mlsteele** [comment]
If you run it twice it gets `SCStellarWrongRevision` from the server and suppresses it. So nothing.

**Pull request merged**